### ISSUE TITLE
release/uwp6.2 build errors -- PackageIndex.json

### ISSUE THREAD

**Issue by zamont** [dotnet/corefx#31619]
The branch release/uwp6.2 is branched off release/2.1, so after branching and getting a successful build, I merged https://github.com/dotnet/corefx/pull/31546 to produce only the Microsoft.Private.CoreFx.UAP package. I ignored the CI build failures because I didn't understand them, but I realize now that the Microsoft.Private.CoreFx.UAP package build fails with 

```
2018-08-06T22:10:57.6409415Z E:\A\_work\165\s\corefx\Tools\Packaging.targets(1150,5): error : File System.IO.Pipes.AccessControl, version 4.0.3.0 was included framework package runtime.win10-arm.Microsoft.Private.CoreFx.UAP/4.6.0-servicing-26806-01 but that version is not considered inbox in package index E:\A\_work\165\s\corefx\pkg\Microsoft.Private.PackageBaseline\packageIndex.json.  Please add it with approriate InboxOn entry for uap10.0.16300 or suppress this message with PermitInbox suppression. [E:\A\_work\165\s\corefx\pkg\Microsoft.Private.CoreFx.UAP\Microsoft.Private.CoreFx.UAP.pkgproj]
2018-08-06T22:10:57.6411028Z E:\A\_work\165\s\corefx\Tools\Packaging.targets(1150,5): error : File System.Memory, version 4.1.0.0 was included framework package runtime.win10-arm.Microsoft.Private.CoreFx.UAP/4.6.0-servicing-26806-01 but that version is not considered inbox in package index E:\A\_work\165\s\corefx\pkg\Microsoft.Private.PackageBaseline\packageIndex.json.  Please add it with approriate InboxOn entry for uap10.0.16300 or suppress this message with PermitInbox suppression. [E:\A\_work\165\s\corefx\pkg\Microsoft.Private.CoreFx.UAP\Microsoft.Private.CoreFx.UAP.pkgproj]
2018-08-06T22:10:57.6412588Z E:\A\_work\165\s\corefx\Tools\Packaging.targets(1150,5): error : File System.Runtime.CompilerServices.Unsafe, version 4.0.4.0 was included framework package runtime.win10-arm.Microsoft.Private.CoreFx.UAP/4.6.0-servicing-26806-01 but that version is not considered inbox in package index E:\A\_work\165\s\corefx\pkg\Microsoft.Private.PackageBaseline\packageIndex.json.  Please add it with approriate InboxOn entry for uap10.0.16300 or suppress this message with PermitInbox suppression. [E:\A\_work\165\s\corefx\pkg\Microsoft.Private.CoreFx.UAP\Microsoft.Private.CoreFx.UAP.pkgproj]
```

@sergiy-k @MichalStrehovsky we intend to ship these assemblies in the metapackage correct? If so, I'll need to update the baseline. @joperezr I think the solution would be to revert your PR https://github.com/dotnet/corefx/pull/29970 . Does that sound right? Again, we branched off release/2.1 so we picked up your specific PR even though it does not exist in the master branch.

cc: @jkotas

**Comment by sergiy-k** [comment]
cc: @weshaggard, @tommcdon 
This is currently a blocking issue with uwp6.2.

**Comment by zamont** [comment]
Merged #31637
